tests/newfeatures/env-deploy.feature

```diff
@@ -9,7 +9,7 @@ Feature: Site Deployment
 
   @vcr site_deploy
   Scenario: Deploy dev to test
-    When I run "terminus env:deploy test --site=[[test_site_name]] --sync-content --note='Deploy test'"
+    When I run "terminus env:deploy [[test_site_name]].test 'Deploy log note' --sync-content"
```

Comment:
@ari-gold Terminus requires it now, but the API does not require it. Why be beholden to the old pattern?
https://github.com/pantheon-systems/titan-mt/blob/master/yggdrasil/yggdrasil_workflows.py#L4448